app/src/composables/use-item/use-item.ts

```diff
@@ -292,7 +292,7 @@ export function useItem(collection: Ref<string>, primaryKey: Ref<string | number
 	}
 
 	function setItemValueToResponse(response: AxiosResponse) {
-		if (response.data.data.collection?.startsWith('directus_')) {
+		if (typeof response.data.data.collection === 'string' && response.data.data.collection.startsWith('directus_')) {
```

Comment:
Right now we're relying on the "collection" field being returned in the data for the item. Shouldn't we instead just use the collection-to-fetch information available in `collection.value` ?

```suggestion
		if (collection.value?.startsWith('directus_')) {
```

Reply:
I don't think we can since the value of `collection.value` is `directus_collections` for all collections in data model (even for non-directus collections).

Reply:
Ohhhhhhh yeah I forgot this is for directus_collections specifically ^^

In that case, lets make it more explicit and add a check for whether or not we're in directus_collections before trying to read the returned collection 👍🏻 

```suggestion
		if (collection.value === 'directus_collections' && response.data.data.collection?.startsWith('directus_')) {
```

Reply:
Makes sense! 👍 Have tested this!